docs/design/marketing-offer-program.md

```diff
@@ -67,54 +67,145 @@ The credit is automatically applied to the account and will have a max limit tha
     id - int
     name -  text
     description - text  
-    credits - integer
+    award_credits_in_cents - integer
+    invitee_credits_in_cents - integer
     redeemable_cap - integer
     num_redeemed - integer
     created_at - timestamp
     expires_at - timestamp
-    award_credit_duration_days - int
+    award_credit_duration_months - int
```

Comment:
Couldn't that lead to calculation problems? Lets say i started on the 17th of Feb. Do i get until the 16th of March?
Is that the same for March - April (3 days more)?

Reply:
We have been going back and forth on this. I think you are right, changed it back to be days